examples/go-code-tutorial/deps.bzl

```diff
@@ -0,0 +1,211 @@
+load("@bazel_gazelle//:deps.bzl", "go_repository")
```

Comment:
seems to me that all these files need commenting so the example can stand on its own to illustrate how things work.

and/or link to the tutorial from all those comments so the two are mutually reinforcing

Reply:
Yah, I noticed this.  Let me update the file more.